tools/sigmut/sigmut.xml

```diff
@@ -0,0 +1,495 @@
+﻿<tool id="SigProfiler" name="SigProfiler" version="1.0">
```

Comment:
Why not linking this `version` to the `@VERSION@` in macros ?

Reply:
Yes, that's totally right.

Reply:
So, go ahead !
```
<tool id="SigProfiler" name="SigProfiler" version="@VERSION@">
```
or something like that.

Reply:
Ping @lgpdv

Reply:
Yes, that's OK. All these changes are already done, but before running the PR I would like to finish the lighter genome (c elegans) approach. It takes a lot of time because I have detected some problems with the authors' code. Locally, I have solved them and I am able to obtain the mut. sign. But, the bioconda package has to corrected...

Reply:
I mean:
the bioconda package has to be corrected...